csharp/developer-guides/using-workers/using-workers.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <RootNamespace>chsarp_example</RootNamespace>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="conductor-csharp" Version="1.1.0" />
```

Comment:
is 1.1.0 pending release? i only see 1.0.15 on NuGet

Reply:
Pending, that's why I have in the description: "WIP - depends on these changes https://github.com/conductor-sdk/conductor-csharp/pull/123"